homework/password-check/validation.cpp

```diff
@@ -1,2 +1,60 @@
 #include "validation.hpp"
-// TODO: Put implementations here
\ No newline at end of file
+// TODO: Put implementations here
+bool isspecial(char ch){
+    if(isalpha(ch)){
+		return false;
+    }
+	if(isdigit(ch)){
+		return false;
+    }
+	else{
+		return true;
+    }
+}
+
+std::string getErrorMessage(ErrorCode errCode){
+    switch(errCode){
+        case Ok: return "Ok";
+        case PasswordNeedsAtLeastNineCharacters: return "Password needs to have at least nine characters";
+        case PasswordNeedsAtLeastOneNumber: return "Password needs to have at least one number";
+        case PasswordNeedsAtLeastOneSpecialCharacter: return "Password needs to have at least one special character";
+        case PasswordNeedsAtLeastOneUppercaseLetter: return "Password needs to have at least one uppercase letter";
+        case PasswordsDoNotMatch: return "Passwords do not match";
+        default: return "Wrong code";
+    }
+}
+
+bool doPasswordsMatch(std::string password1, std::string password2){
+    if(password1 == password2){
+        return true;
+    }
+    else{
+        return false;
+    }  
```

Comment:
Add space after `if`, `for`, `while` etc...

When you use return inside `if` you don't need `if else` or `else`. Just write `if (sth) { return blabla; }`